apollo-router/src/plugins/authorization/mod.rs

```diff
@@ -556,8 +556,10 @@ impl Plugin for AuthorizationPlugin {
                         Ok(ControlFlow::Continue(request))
                     } else {
                         // This is a metric and will not appear in the logs
-                        tracing::info!(
-                            monotonic_counter.apollo_require_authentication_failure_count = 1u64,
+                        u64_counter!(
+                            "apollo_require_authentication_failure_count",
+                            "Number of unauthenticated requests (deprecated)",
```

Comment:
I marked this as deprecated because the naming doesn't seem to follow convention, but I don't know if we have a proper alternative for this? We have `apollo.router.operations.authorization` below but I'm not sure it reports the same thing.

Reply:
I think that's the right thing to do. cc @BrynCooke you probably have more context

Reply:
For now let's not mark these as deprecated. We will then circle back and fix the names.

Reply:
We're going to have to add a new metric with the desired name.... 🤔

Reply:
I think this can just go. The existing `http.server.request.duration` metric will have a status code on it already, and people can just look for `StatusCode::UNAUTHORIZED`.